packages/deployment/app/cloudformation.template.yml

```diff
@@ -1202,13 +1202,125 @@ Resources:
     Properties:
       Cluster:
         Ref: CumulusECSCluster
-      DesiredCount: {{# if this.count}}{{this.count}}{{ else }} 0 {{/if}}
+      DesiredCount: {{# if this.count}}{{this.count}}{{ else }}0{{/if}}
       TaskDefinition:
         Ref: {{@key}}TaskDefinition
       DeploymentConfiguration:
         MaximumPercent: 100
         MinimumHealthyPercent: 0
 
+  {{! If minTasks and maxTasks are not configured for this service, do not configure autoscaling }}
+  {{# if this.minTasks }}{{# if this.maxTasks }}
+
+  {{@key}}ECSServiceScalableTarget:
+    Type: AWS::ApplicationAutoScaling::ScalableTarget
+    Properties:
+      MinCapacity: {{this.minTasks}}
+      MaxCapacity: {{this.maxTasks}}
+      ResourceId:
+        Fn::Sub:
+          - service/${CumulusECSCluster}/${ServiceName}
+          - ServiceName:
+              Fn::GetAtt:
+                - "{{@key}}ECSService"
+                - Name
+      RoleARN: {{../this.iams.scalingRoleArn}}
+      ScalableDimension: ecs:service:DesiredCount
+      ServiceNamespace: ecs
+
+  {{# if this.scaleInAdjustmentPercent }}
```

Comment:
I feel like this is somewhat interesting. Do we want to give them the option of scaling up without scaling down? If we do, I think the interaction/configuration should be specifically documented somewhere.

The implementation here makes it look like a user can not specify  a `scaleInAdjustmentPercent` and then their tasks will _never_ be scaled in.

Reply:
See https://github.com/nasa/cumulus/pull/942#issuecomment-484949251